celery/app/defaults.py

```diff
@@ -290,6 +290,9 @@ def __repr__(self):
         __old__=OLD_NS_WORKER,
         agent=Option(None, type='string'),
         autoscaler=Option('celery.worker.autoscale:Autoscaler'),
+        cancel_long_running_tasks_on_connection_loss=Option(
```

Comment:
any shorter name in mind?

Reply:
@thedrow and I spoke about this a bit out of band and it sounded to me like being more explicit about what the option does is probably best. I could only imaging this being called something like `connection_loss_strategy` and having an enum of different behaviours (which would be like `DEFAULT_STRATEGY`, `IGNORE_EVENT` and `CANCEL_TASKS` or something like that).

I think since there are only two strategies right now, this long and more explicit name is probably okay.